### DISCUSSION TITLE
cafcfd0d/

### DISCUSSION THREAD

**Discussion by giscus** [james1BadCreeper/james1BadCreeper.github.io#17] [Announcements]
# cafcfd0d/

动态规划（Dynamic Programming，DP）是运筹学的一个分支，是求解决策过程最优化的过程。20 世纪 50 年代初，美国数学家贝尔曼（R.Bellman）等人在研究多阶段决策过程的优化问题时，提出了著名的最优化原理，从而创立了动态规划。动态规划问世以来，在经济管理、生产调度、工程技术和最优控制等方... - james1BadCreeper - james1 的博客

https://james1badcreeper.github.io/cafcfd0d/?keyword=%E5%8A%A8%E6%80%81%E8%A7%84%E5%88%92

**Comment by lyrenius** [comment]
博主好厉害!!!1
